hub/api/dataset_utils.py

```diff
@@ -0,0 +1,47 @@
+from typing import Union
+from hub.codec import PngCodec, Lz4, NumPy, Zstd, JpegCodec, WebPCodec
+from hub.codec import BaseImgCodec, BaseNumCodec
+
+compression_map = {
+    PngCodec().__name__: PngCodec,
+    JpegCodec().__name__: JpegCodec,
+    WebPCodec().__name__: WebPCodec,
+    Lz4().__name__: Lz4,
+    Zstd().__name__: Zstd,
+    NumPy().__name__: NumPy,
+}
+
+
+def _get_compressor(
```

Comment:
if this method is being used from other files you should not put a `_` in front of it's name